Fix some grammar/punctuation + rewrite stuff (#605)
hyprwm/hyprland-wiki: pages/Getting Started/Installation.md

## Installation

Installing Hyprland is very easy. Simply install it with your package manager 

> (if the pkg is available for Hyprland) or install/build it yourself.

Reviewer: ```suggestion
(if there is a Hyprland package available) or install/build it yourself.
```
